script/generateUtilsDoc.sh

```diff
@@ -14,8 +14,26 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+NEW_DEPLOY_NEEDED=0
+
+if [ -z $GH_TOKEN  ]
```

Comment:
This has to be [GITHUB_TOKEN](https://docs.travis-ci.com/user/deployment/pages/)

Reply:
adapted